src/assets/data/integrations/IntegrationList.js

```diff
@@ -196,6 +199,24 @@ const IntegrationList = [
     status: "inProduction",
     category: "Service Mesh",
   },
+  {
+    name: "Prometheus",
+    img: prometheus,
+    status: "InProgress",
```

Comment:
The status is defined in multiple ways somewhere it is `inProgress` while somewhere it is `InProgress`. Please make it consistent with `InProgress`

Reply:
So , the progresss status is defined by "InProgress" and production by "inProduction" . Problem here is with status of github so can you please confirm is it in progress or production.

Reply:
InProgress

Reply:
Done